Library/Formula/sqtop.rb

```diff
@@ -0,0 +1,15 @@
+class Sqtop < Formula
+  homepage "https://github.com/paleg/sqtop"
+  url "https://github.com/paleg/sqtop/archive/v2015-02-08.tar.gz"
+  version "2015-02-08"
+  sha1 "379a97e0190f3da39e2d67096955c40217b39ae5"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    assert_match "#{version}", shell_output("#{bin}/sqtop --help | grep version | cut -d \" \" -f 2")
```

Comment:
I guess no, this program is just a client and requires fully working and configured [squid](http://www.squid-cache.org/) server to do useful work.
Without it all launches will fail.